plugin-server/tests/main/db.test.ts

```diff
@@ -1026,4 +1026,19 @@ describe('DB', () => {
             )
         })
     })
+
+    describe('addEventToBuffer', () => {
+        test('inserts event correctly', async () => {
```

Comment:
Most tests use `it`.

Rule of thumb:
- Use `it` for unit tests
- Use `test` or `it` for integration tests